### ISSUE TITLE
checkstyleMain task is failing with Gradle 8.0 upgrade

### ISSUE THREAD

**Issue by cdlischynski** [xvik/gradle-quality-plugin#77]
Looks like it may have something to do with the getOutputLocation return type being changed from Provider to Property.
https://docs.gradle.org/current/userguide/upgrading_version_7.html#report_getoutputlocation_return_type_changed_from_provider_to_property

Here is my stack trace: 
```
Caused by: java.lang.NoSuchMethodError: 'org.gradle.api.provider.Provider org.gradle.api.reporting.Report.getOutputLocation()'
        at ru.vyarus.gradle.plugin.quality.report.ReportUtils.getReportFile(ReportUtils.groovy:115)
        at ru.vyarus.gradle.plugin.quality.report.ReportUtils$getReportFile.call(Unknown Source)
        at ru.vyarus.gradle.plugin.quality.report.CheckstyleReporter.report(CheckstyleReporter.groovy:27)
        at ru.vyarus.gradle.plugin.quality.report.CheckstyleReporter.report(CheckstyleReporter.groovy)
        at ru.vyarus.gradle.plugin.quality.QualityPlugin$_applyReporter_closure10.doCall(QualityPlugin.groovy:417)
```

https://github.com/xvik/gradle-quality-plugin/blob/master/src/main/groovy/ru/vyarus/gradle/plugin/quality/report/ReportUtils.groovy#L115

**Comment by xvik** [comment]
Thank you! I'll look

**Comment by xvik** [comment]
[released 4.9.0](https://github.com/xvik/gradle-quality-plugin/releases/tag/4.9.0)
